releases/cloud.md

```diff
@@ -16,6 +16,20 @@ Get future release notes emailed to you:
 
 {% include releases/current-cloud-version.md %}
 
+## January 9, 2023
+
+<h3> Console changes </h3>
+
+- {{ site.data.products.dedicated }} clusters now have access to the [**Metrics** page](../v22.2/ui-custom-chart-debug-page.html#available-metrics).
```

Comment:
@gemma-shay we're still gating this behind a feature flag and have not opened it up for all users yet. I recommend we cut this for the Jan release note.